### PR TITLE
Games: Add -removeplayers command

### PR DESCRIPTION
**Pull request by Kalest3**
i've been using Expecto Botronum's leaderboard commands a lot in the Portugues room, and i think a command that removes players from the leaderboard would be helpful.

### CODE REVIEW

File: modules/games.py [hunk 1]
```diff
@@ -149,6 +150,38 @@ def addPoints(self, message: core.BotMessage) -> None:
 
         return message.respond("Points added!")
 
+    def removePlayers(self, message: core.BotMessage) -> None:
+        """Remove users from the minigame leaderboard
+
+        Arguments:
+            message {Message} -- the Message object that invoked the command
+        """
+        if not message.room: return message.respond("You can only remove players in a room.")
```
[the remark this answers is not in the document]
Comment (Kalest3) [reply modules/games.py:159]: ok

File: modules/games.py [hunk 2]
```diff
@@ -149,6 +150,38 @@ def addPoints(self, message: core.BotMessage) -> None:
...
+        if not message.sender.can("hostgame", message.room): return message.respond("Permission denied.")
+
+        if len(message.arguments) < 2:
+            return message.respond(
+                f"Usage: ``{config.commandCharacter}removeplayers [comma-separated list of users]``."
+            )
+        usernames = message.arguments[1:]
+        usersOnLB = []
+        usersNotOnLB = []
+if message.room.id not in self.minigamePoints.keys(): return message.respond("There are no scores.")
```
Comment (AnnikaCodes) [inline modules/games.py:169]:
```suggestion
        if message.room.id not in self.minigamePoints.keys(): return message.respond("There are no scores.")
```

Comment (Kalest3) [reply modules/games.py:169]: I did some tests on a local server and this line results in a websocket error if done in the bot's PM

File: modules/games.py [hunk 3]
```diff
@@ -149,6 +150,40 @@ def addPoints(self, message: core.BotMessage) -> None:
...
+    def removePlayers(self, message: core.BotMessage) -> None:
+        """Remove users from the minigame leaderboard
+
+        Arguments:
+            message {Message} -- the Message object that invoked the command
+        """
+        PM = False
+        room = message.room
+        if not room:
+            if len(message.arguments) < 3: return message.respond("You must specify a room and at least an user.")
```
Comment (AnnikaCodes) [inline modules/games.py:162]:
```suggestion
            if len(message.arguments) < 3: return message.respond("You must specify a room and at least one user.")
```

File: modules/games.py (hunk 2 above)
Comment (AnnikaCodes) [reply modules/games.py]: uhhh, that shouldn't be happening? Maybe my websocket library is broken; I wrote it when I was a much worse programmer than I am now.

Comment (AnnikaCodes) [reply modules/games.py]: In any case, you should leave the line in - if responding to PMs is broken, that's going to cause issues regardless of this PR.

Comment (Kalest3) [reply modules/games.py]: well, anyway, the room.id worked

**Comment by AnnikaCodes** [comment]
This is good enough to merge, thanks!

**Pull request merged**